### ISSUE TITLE
JGecko U Support (A0 Range Kernel Mapping)

### ISSUE THREAD

**Issue by BullyWiiPlaza** [dimok789/loadiine_gx2#97]
@dimok789 [Can you please enable the A0 kernel mapping](https://github.com/dimok789/loadiine_gx2/blob/master/installer/launcher.c#L172#L173) so that [CosmoCortney's real time cheat code handler](https://github.com/CosmoCortney/PPC-Cheat-Code-Handler) can be injected using [JGecko U without crashing](http://gbatemp.net/threads/jgecko-u-setup-guide.426292/)?

Thanks :+1:

**Comment by dimok789** [comment]
Nah. Enabling that mapping is gonna break some games compatibility.

I can suggest you to add a command to write that mapping over pygecko and you guys add that in your code handler. What do you think?

**Comment by dimok789** [comment]
I can also suggest an any position write functionality too. Than you dont need the A0 mapping and can access the memory directly.

**Comment by BullyWiiPlaza** [comment]
Yeah, whatever it takes to make it work
